src/main/scala/no/ndla/oembedproxy/service/ProviderService.scala

```diff
@@ -79,7 +79,20 @@ trait ProviderService {
       OEmbedEndpoint(Some(H5PApprovedUrls), Some(s"${OEmbedProxyProperties.NdlaH5POembedProvider}/oembed"), None, None)
     val H5PProvider = OEmbedProvider("H5P", OEmbedProxyProperties.NdlaH5POembedProvider, List(H5PEndpoint))
 
-    val TedApprovedUrls = List("https://www.ted.com/talks/*")
+    val TedApprovedUrls = List(
+      "https://www.ted.com/talks/*",
```

Comment:
Må vi støtte både http og https? Og kunne ikkje dette vore skreve som "//www.ted.com/talks/*"? Eller støtter ikkje biblioteket det?

Reply:
Kunne vi eventuellt hatt //*.ted.com og //ted.com som gyldige urler?

Reply:
Usikker, i første omgang så tenkte jeg å ha `*.ted.com/talks/*` som whitelist, men så påpekte Jonas sikkerhetshull ved denne løsningen og da valgte jeg å gå full eksplisitt. Angående http vs https så tenkte jeg just in case noen skriver url'en feil så kan vi være ekstra snile og godkjenne det (blir gjort om til https uansett når den embedes).

Reply:
Du tenkte vel å ha `*ted.com` som er et mye større sikkerhetshull enn `*.ted.com` (Siden da stoler vi bare på alle `ted.com` subdomener og ikke alle domener som slutter på `ted.com`).
Denne strengen er bare en regex som sjekkes mot url'en brukeren sender inn så vi kan støtte akkurat hva vi har lyst til så vidt jeg kan se.

Reply:
Joda, men spørsmålet er om du kan ha '//www.ted.com/talks/*' som godkjent url, altså uten http eller https. Kan du ikkje i det minste teste det?

Reply:
Sjekket nettopp og fant ut at `//www.ted.com/talks/*` vil ikke fungere med vår implementasjon av embed i frontenden. Når man skal sette inn en ressurs fra lenke, sjekkes da lenken mot oembed-proxy for å finne om den er whitelisted (og her ville `//www.ted.com/talks/*` gått), men når lenka skal vises som embed gjøres det en oppslag til som da baserer seg på url'en vi får fra oembed-proxy når vi setter lenka inn i artikkelen, som er på formatet `https://www.embed.ted.com/talks/blablabla` og det er her det knekker hvis vi whitelister bare på denne måten: `//www.ted.com/talks/*` siden vi mangler https. Som Jonas skrev så er dette bare en string vi setter inn her som da sjekkes med regex mot url'en fra bruker, `//` blir ikke betraktet som "både http og https", men som en streng.

Reply:
En annen sak er å whiteliste `*//ted.com/talks/*` men vet ikke hvor lurt det er